### DISCUSSION TITLE
Which flags used to build Swoole (php8.2-swole)?

### DISCUSSION THREAD

**Discussion by andrew-demb** [oerdnj/deb.sury.org#2064] [Q&A] [answered]
Swoole provides custom compile options - https://github.com/swoole/swoole-src/tree/v5.1.1?tab=readme-ov-file#extra-compiler-configurations

For example: `--enable-swoole-curl`

What flags are used to build a package for the Swoole extension (for example: `php8.2-swoole`)?

**Comment by oerdnj** [comment, marked as answer]
You can check this yourself by adding `deb-src` source to [sources.list](https://manpages.debian.org/bookworm/apt/sources.list.5.en.html) and then running `apt source php8.2-swoole` and looking into `debian/rules`.
